libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -154,6 +154,7 @@ class AP_SerialManager {
         SerialProtocol_FETtecOneWire = 38,
         SerialProtocol_Torqeedo = 39,
         SerialProtocol_AIS = 40,
+        SerialProtocol_ROS2 = 41,
```

Comment:
@peterbarker You had mentioned to limit the scope of this to serial only. One thing I am wondering is whether this should be called ROS2, or more appropriately name it based on the protocol, which would be `SerialProtocol_DDS_XRCE` to match the official OMG standard? ROS2 is not a protocol, it's a framework.  
https://www.omg.org/spec/DDS-XRCE/

Reply:
I renamed in in 4446a11d5e02e1ae19d8b0d86bbb6a6d447d72ad